packages/core/src/dialects/sqlite/query-generator-typescript.ts

```diff
@@ -196,4 +197,19 @@ export class SqliteQueryGeneratorTypeScript extends AbstractQueryGenerator {
 
     return fragment;
   }
+
+  setIsolationLevelQuery(isolationLevel: IsolationLevel): string {
+    switch (isolationLevel) {
+      case IsolationLevel.REPEATABLE_READ:
+        return '-- SQLite is not able to choose the isolation level REPEATABLE READ.';
```

Comment:
shouldn't this throw? Nothing new but still super strange. TODO?

Reply:
I've updated this to throw now.

Reply:
Thank you